parameters: 
  InspireGripper: 
    joint_names: ["left_finger_1_joint", "left_finger_2_joint", "right_finger_1_joint", "right_finger_2_joint"]
    joint_state: [0, 0, 0, 0]
    action_points: 1
    action_positions: 3
    
  AirpowereGripper:
    joint_names: ["pickup_gripper_joint"]
    joint_state: [0]
    action_points: 1
    action_positions: 0
